e2e/test-default-route1.sh

```diff
@@ -7,7 +7,13 @@ kubectl create -f yamls/default-route1.yml
 kubectl wait --for=condition=ready -l app=default-route1 --timeout=300s pod
 
 echo "check default-route-worker1 interface: net1"
-kubectl exec default-route-worker1 -- ip a show dev net1
+net=$(kubectl exec default-route-worker1 -- ip a show dev net1)
```

Comment:
the variable `net` seems not to used anywhere. could you please let me know why `net` is required?

Reply:
ok, echo $net

Reply:
We need kubectl output even though error code is non 0, for troubleshooting. How about to keep `kubectl exec default-route-worker1 -- ip a show dev net1`?

Reply:
> We need kubectl output even though error code is non 0, for troubleshooting. How about to keep `kubectl exec default-route-worker1 -- ip a show dev net1`?
ok,updated it , kubectl output will be printed
![image](https://user-images.githubusercontent.com/31728060/204736084-57035254-50a6-4611-8b71-e17fde6fe987.png)

Reply:
This code still seems to be redundant. how about that?

```
kubectl exec default-route-worker1 -- ip a show dev net1
if [ $? -ne 0 ];then
       exit 1
fi
```